speech/cloud-client/pom.xml

```diff
@@ -0,0 +1,59 @@
+<!--
```

Comment:
Really? The test definitely ran during the [travis build](https://travis-ci.org/GoogleCloudPlatform/java-docs-samples/builds/197720320), or are you refereing to something else?